### ISSUE TITLE
Implement regex-capable set plugin

### ISSUE THREAD

**Issue by CapacitorSet** [Telegram-Bot-Node/Nikoro#37]
eg.

```
A: /set foo bar
Bot: Set.
A: lalala foo lalala
Bot: bar

----

A: /set /fo+/i bar
Bot: Set.
A: lalala Foooooooo lalala
Bot: bar

----

A: /set /f(o+)/ bar
Bot: Set.
A: Foooo
A: foo
Bot: oo
```

**Comment by CapacitorSet** [comment]
If regexes are used, consider using a linear-time library: https://swtch.com/~rsc/regexp/regexp1.html

For example, [this](https://github.com/substack/safe-regex) could be a good library to check regular expressions against.

**Comment by crisbal** [comment]
The only thing I am worried is about performance, would you match any message with any regex?

(We already do this in the master/dev version and I am worried about it too)

**Comment by CapacitorSet** [comment]
Fixed in 3ef1d780c2393232fa0b0e91553e06f6dcb67885.
